apps/vanilla/src/pages/index.html

```diff
@@ -1,16 +1,119 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./index.css" />
-    <title>Vite App</title>
-  </head>
-  <body class="default-layout">
-    <h1 class="title">Main page ⚡</h1>
-    <div class="links">
-      <a href="example/">Go to example page</a>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./index.css" />
+  <title>Vite App</title>
+</head>
+
+<body class="default-layout">
+  <!-- Header -->
+  <header class="books-header text-logo">📚 Books app</header>
+  <main class="catalog">
+    <!-- Books catalog -->
+    <article class="catalog__content">
+      <section class="books-category">
+        <div class="books-category__sticky-container">
+          <h2 class="text-subtitle">2020 year</h2>
+        </div>
+        <ul class="books-list">
+          <li class="book-card">
+            <img class="book-card__img" src="./assets/images/image 3.png" alt="Refactoring UI book" />
```

Comment:
Let's either add `book` to all of the alt texts, or omit it from here to be consistent
```suggestion
            <img class="book-card__img" src="./assets/images/image 3.png" alt="Refactoring UI" />
```